components/my-account/Lists/ListTable/ListTable.tsx

```diff
@@ -0,0 +1,171 @@
+import React, { useState } from 'react'
+
+import { Delete, Edit, FolderCopySharp, MoreVert } from '@mui/icons-material'
+import {
+  Box,
+  Button,
+  IconButton,
+  Menu,
+  MenuItem,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  useMediaQuery,
+  useTheme,
+} from '@mui/material'
+import { useTranslation } from 'next-i18next'
+
+import { styles } from '@/components/my-account/Lists/ListTable/ListTable.style'
+import formatDate from '@/lib/helpers/formatDate'
+
+import { CrWishlist, Maybe } from '@/lib/gql/types'
+
+interface ListTableProps {
+  rows: CrWishlist[]
+  onDeleteList: (param: Maybe<string> | undefined) => void
+  onCopyList: (param: Maybe<string> | undefined) => void
+  onEditList: (param: Maybe<string> | undefined) => void
+  isLoading: boolean
+}
+
+const ListTable = (props: ListTableProps) => {
+  const { rows, onDeleteList, onCopyList, onEditList, isLoading } = props
+  // state for mobile menu
+  const [anchorEl, setAnchorEL] = useState<HTMLElement | null>(null)
+
+  const { t } = useTranslation('common')
+  const theme = useTheme()
+  const mdScreen = useMediaQuery(theme.breakpoints.up('md'))
+
+  const options = [
+    { name: 'Edit', onClick: onEditList },
+    { name: 'Add list items to cart', onClick: () => alert('Work in progress') },
```

Comment:
Why are we using alert ?

Reply:
this code is temporary, once this PR is merge, will remove the alert with the function required